helpers/hpytest.py

```diff
@@ -81,3 +82,192 @@ def pytest_clean(dir_name: str, preview: bool = False) -> None:
     # Show after cleaning.
     file_names = _pytest_show_artifacts(dir_name, tag="After cleaning")
     hdbg.dassert_eq(len(file_names), 0)
+
+
+# #############################################################################
+# Colors
+# #############################################################################
+
+
+class Colors:
+    """
+    ANSI color codes for terminal output.
+    """
+
+    RED = "\033[91m"
```

Comment:
I think we have this in hlogger or hprint. Ok with duplicating it for now, but can you pls double check if we can merge (even in a later PR)?